### PR TITLE
logo

### PR DESCRIPTION
**Pull request by lzehl**

### CODE REVIEW

**Review by Peyman-N — APPROVED** [review]

**Comment by Peyman-N** [comment]
This should be changed to reflect the new name.

File: img/openMINDS_MRI_logo_dark.png
Comment (Peyman-N) [inline img/openMINDS_MRI_logo_dark.png:1]: This should be changed to reflect the new name.

**Review by Peyman-N — CHANGES_REQUESTED** [review]

**Review by Peyman-N — CHANGES_REQUESTED** [review]
This should be changed to reflect the new name.

**Comment by lzehl** [comment]
@UlrikeS91 would you be able to take over? I would suggest to reduce the name for the logo to "nimaging" or only "imaging" (if the long name "neuroimaging" does not fit)

**Comment by UlrikeS91** [comment]
> @UlrikeS91 would you be able to take over? I would suggest to reduce the name for the logo to "nimaging" or only "imaging" (if the long name "neuroimaging" does not fit)

Yes, I can fix this. Tried a few options now and none of your suggestions fit.  The best alternative I see is: "nImg" (similar to specimenPrep, where we use sPrep). I'll fix it with this abbreviation. If you're absolutely against it, I need a better suggestion that is no longer then 5 characters 😉

**Comment by lzehl** [comment]
@UlrikeS91 opened a new PR for the updated logos; so I'm closing this one now.

**Pull request closed without merging**